src/index.js

```diff
@@ -1,167 +1,56 @@
-// @flow
+require("dotenv").config();
```

Comment:
Should not be necessary here

Reply:
usually we only use `dotenv` in development:
https://github.com/orbiting/backends/blob/24b1b9e78de911a054e78eb43e93eacc04ba4c8d/packages/env/index.js#L8